app/public/js/profile.js

```diff
@@ -106,12 +106,16 @@ function getCountryElement()
 
 function updateRegionOptions()
 {
-	var country_id = parseInt(getCountryElement().val());
-	var $datalist = $('#regions');
-	$datalist.empty();
+	let country_id = parseInt(getCountryElement().val());
+	let home_region = $('#home_region');
+	home_region.empty();
 	regions.forEach(function(region) {
 		if ( region.country_id === country_id )
-			$datalist.append($('<option />').text(region.name));
+			if (home_region.data('value') === region.name) {
+                home_region.append($('<option />').val(region.name).attr('selected', 'selected').text(region.name));
```

Comment:
Fix indentation.  The body of the if-statement should be indented more than the 'if' line.